locale/en/blog/vulnerability/openssl-november-2022.md

```diff
@@ -21,8 +21,6 @@ will release OpenSSL 3.0.7 on the 1th of
 November 2022 between 1300-1700 UTC. The release will fix security defects on which
 the _highest_ severity issue is CRITICAL.
```

Comment:
Maybe the link was supposed to be part of this sentence. Just added the anchor to skip to the section that talks about the severity
```suggestion
the _highest_ severity issue is [CRITICAL](https://www.openssl.org/policies/general/security-policy.html#issue-severity).
```

Reply:
Actually, I think we should adjust from CRITICAL to HIGH. See: https://mta.openssl.org/pipermail/openssl-announce/2022-November/000243.html

Reply:
Done